src/components/CreateRules/UploadRulesSummary.vue

```diff
@@ -69,12 +64,36 @@ export default class UploadRulesSummary extends Mixins(DocumentMixin) {
 }
 
 .upload-rules-error-message {
-  padding-top: 1.25rem;
+  padding-top: 30px;
+  padding-bottom: 30px;
   padding-left: 1.25rem;
   color: $app-blue;
+
+  .error-text {
+    margin-right: 4px;
+  }
+}
+
+.upload-rules-success-message {
+  padding-top: 30px;
+  padding-bottom: 30px;
+  padding-left: 1.25rem;
+  color: $gray7;
+
+  .upload-success-chk {
+    margin-right: 9px;
+    font-size: 1.5rem;
+    color: #2e8540;
+  }
+
+  #file-name-col {
+    margin-top: 2px;
+    margin-left: -50px;
+  }
```

Comment:
With Padding and Margins, we've traditionally tried to use the vue helpers in the template. 
ie `class= mr-1` as an example for margin right. It's roughly 4px increments. 
If you achieve the same thing using those helper classes it really minimizes the code in the style blocks.
Your call but would be nice for consistency :)

Reply:
will take a look and maybe ping you when i get into it.

Reply:
It's way easier to define/adjust layout when padding/margins is declared in the template (using the Vuetify helper classes).

Otherwise, a developer has to go look up all the possible places a class could have rules.

Reply:
updated such that most spacing is done in the template with vuetify helper classes.  i like it and definitely a lot cleaner.

Reply:
LGTM